SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestCharacterMigration/SpatialTestCharacterMigration.cpp

```diff
@@ -157,6 +161,10 @@ void ASpatialTestCharacterMigration::PrepareTest()
 			{
 				PlayerCharacter->SetActorLocation(FVector(0.0f, -25.0f, 50.0f));
 			}
+			else
+			{
+				PlayerCharacter->SetActorLocation(FVector(0.0f, 50.0f * FlowControllerId, 50.0f));
```

Comment:
Why special case the first flow controller?

Reply:
I've removed this step as this test was designed for one client and is flakey with multiple clients. Instead, I set detect on the WorldSettings of the corresponding test map to ensure that only one client runs.